nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/state/provenance-event-listing/provenance-event-listing.effects.ts

```diff
@@ -58,10 +63,14 @@ export class ProvenanceEventListingEffects {
                             response
                         })
                     ),
-                    catchError((error) =>
+                    catchError(() =>
                         of(
-                            ProvenanceEventListingActions.provenanceApiError({
-                                error: error.error
+                            ProvenanceEventListingActions.loadProvenanceOptionsSuccess({
+                                response: {
+                                    provenanceOptions: {
+                                        searchableFields: []
+                                    }
+                                }
```

Comment:
If the call to `getSearchOptions` fails, we end up with a partially complete search dialog where only the date fields are available and the buttons aren't at the bottom.
<img width="774" alt="Screenshot 2024-02-13 at 1 03 18 PM" src="https://github.com/apache/nifi/assets/713866/fec72bea-8abb-4572-b859-efd7c70aad64">

I'm not certain on what to do in this case. Does a banner error make sense? Should we just close the dialog and throw up a snackbar error?
If we can;t do anything with the dialog in the state it is in, we should probably close it and go with a snackbar error. However, It seems that you can still submit a valid query from it. So, maybe at banner error is better. We should also be sure the buttons are snapped to the bottom too.

Reply:
This behavior was from the current NiFi UI. The searchable fields are not loaded when the dialog opens, but rather once when the page loads. I'll update the dialog to indicate when there are no searchable fields available.